src/Overlay.js

```diff
@@ -17,30 +30,60 @@ class Overlay extends React.Component {
       , target
       , placement
       , rootClose
+      , children
+      , animation: Transition
       , ...props } = this.props;
 
-    let positionedChild = (
-      <Position {...{ container, containerPadding, target, placement }}>
-        { this.props.children }
-      </Position>
-    );
+    let child = null;
 
-    if (rootClose) {
-      positionedChild = (
-        <RootCloseWrapper onRootClose={this.props.onHide}>
-          { positionedChild }
-        </RootCloseWrapper>
+    if ( Transition === true ){
+      Transition = Fade;
```

Comment:
Because most of the time it will be a component (except in this case where it seems to be a boolean, only to be replaced by the `Fade` component however) and since it's used in JSX below, if it wasn't capitalized React would think it is a regular DOM element and not a React component.